src/main/app/forms/models/soleTraderDetails.ts

```diff
@@ -0,0 +1,57 @@
+import { MaxLength } from 'class-validator'
+import { PartyDetails } from './partyDetails'
+import DateOfBirth from 'forms/models/dateOfBirth'
+import { PartyType } from 'forms/models/partyType'
+
+export class ValidationErrors {
+  static readonly ORGANISATION_NAME_TOO_LONG: string = 'Enter organization name no longer than $constraint1 characters'
```

Comment:
is this from ben?

Reply:
no, from prototype, Think better if we do it a content sweep the way we did it last time.

Reply:
no ask ben